platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -1396,4 +1433,15 @@ def set_power_override(self, power_override, power_set):
             A boolean, True if power-override and power_set are set successfully,
             False if not
         """
-        return NotImplementedError
+        if self.sfp_type == QSFP_TYPE:
+            power_override_bit = 0
+            if power_override:
+                power_override_bit |= 1 << MCIA_ADDR_POWER_OVERRIDE_POR_BIT
+            power_set_bit = 0
+            if power_set:
+                power_set_bit |= 1 << MCIA_ADDR_POWER_OVERRIDE_PS_BIT
+            power_override_mask = 1 << MCIA_ADDR_POWER_OVERRIDE_PS_BIT | 1 << MCIA_ADDR_POWER_OVERRIDE_POR_BIT
+
+            return self._write_i2c_via_mcia(0, 0x50, MCIA_ADDR_POWER_OVERRIDE, power_set_bit|power_override_bit, power_override_mask)
+        else:
```

Comment:
suggest to use an explicite case for 'SFP',  currently, there is another type of optical module  'OSFP' though we don't support yet.

Reply:
since the function returns NotImplementedError for both SFP and OSFP, we don't handle SFP explicitly for now.